index.html

```diff
@@ -379,7 +380,7 @@ <h2>Glossary</h2>
 <a href="#links"><abbr title="Back to Alphabetical links">&#x2191;</abbr></a>
 
 <div class="letter_anchor" id="o">O</div>
-<p><dfn class="lint-ignore export">Othographic syllable</dfn> A [=typographic character unit=] that includes more than one [=grapheme cluster=]. These are most commonly found in Brahmi-derived scripts, such as Devanagari, or Balinese, when forming conjuncts or stacks. This term is used but not defined in the Unicode Standard: it is defined here for use by W3C documents, such as layout requirements.</p>
+<p><dfn class="lint-ignore export">Othographic syllable</dfn> A [=typographic character unit=] that includes one or more [=grapheme clusters=]. These are most commonly found in Brahmi-derived scripts, such as Devanagari, or Balinese, when forming conjuncts or stacks. They commonly demarcate sequences of characters that are different from phonetic syllables, and they may even span word boundaries. See also the &lt;a href=&quot;https://unicode.org/glossary/#orthographic_syllable&quot; target=&quot;_blank&quot;&gt;definition&lt;/a&gt; in the Unicode Standard.</p>
```

Comment:
Consider parenthesizing "such as Devanagari" instead of using a wayward comma:

> ... Brahmi-derived scripts (such as Devanagari) or Balinese, when...

---

Note: It is a lot easier to work with text in github when lines are manually broken. In order to use a suggestion, here I would have to retype the entire paragraph when mostly what I want is a specific couple of lines.

Reply:
Fixed.